pom.xml

```diff
@@ -437,13 +435,7 @@
                 </executions>
                 <configuration>
                     <instructions>
-                        <Import-Package><![CDATA[
-                            android.util;resolution:=optional,
-                            org.bouncycastle.jce;resolution:=optional,
-                            org.bouncycastle.jce.spec;resolution:=optional,
-                            *
-                        ]]>
-                        </Import-Package>
+                        <_include>-bnd.bnd</_include>
```

Comment:
Since I've never used this, I just want to confirm: 

Is this really `<_include>` (with an underscore) and not `<include>` ?  As well as `-bnd.bnd` and not `bnd.bnd` ?

Reply:
Yes. Not sure why they use the "_". The - in front of bnd.bnd means that the file is optional. So we can start without bnd.bnd files in the subprojects and add them only when something needs to be customized.

Reply:
Thanks!